### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by WimDeMeester** [WimDeMeester/eFinder#46] [General]
Hi @mrosseel, hi @AstroKeith,

I think that the latest version from the main branch works the same as the latest official release.  I think we can make a new release now.  Are there things that still need to be implemented before I make a new release?

**Comment by mrosseel** [comment]
I'm still on my big bang refactor so no new stuff from my side.
BTW: I made my repo public at https://github.com/mrosseel/eFinder

**Comment by WimDeMeester** [comment]
Yesterday, I found a problem in the non-GUI version of the eFinder software.  I got a problem with capture.axy, I'll try to reproduce this and fix the problem before I make a new release.
